### PR TITLE
Works and about myself sections

### PR DESCRIPTION
**Pull request by As1imwe-Mark**
I added a works and About myself sections to my previous Portifolio mobile Website the are aimed at showcasing the different skills I obtained throughout the days at microverse like the use of linters, great github work flow and many others.

I implemented the following requirements
General requirements
Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config):
Linter should be set up correctly for this repository.
Linter result should be green.
There should be no custom changes to linter config.
Make sure that you used correct [GitHub Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md):
README file should be descriptive.
Commit messages should be meaningful.
Pull Request should have a descriptive title telling you what changes are inside.
Pull Request should have a short summary describing introduced changes in more detail.
Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md):
Branch should describe a feature or action (e.g. 'created_sidebar' or 'added_comments_section').
Pull Request is made from 'feature branch' into the main branch.

Make sure the content of the "About myself" section is aligned with flexbox.

In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.

Please use ul and li for categories in "Work" section. Html, CSS, and JavaScript should be list items.

Please use ul and li for categories in "About myself" section. Languages, Frameworks, and Skills should be list items.

You don't need to implement the detail modal window that requires JavaScript.

You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Implement the button interactions (enable, hover, etc.).

### CODE REVIEW

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
### Hi @As1imwe-Mark      ,

Good job implementing most of the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there!



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- No linters errors ✔✔

## (Changes Required) Aspects to improve: ♻️
- [ ] Please check the inline comments below.

## [Optional] suggestions:

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md
Comment (rica213) [inline README.md:1]: Great documentation Mark 🥳

**Review by rica213 — APPROVED** [review]
# 🟢 STATUS: APPROVED

Hi Mark (@As1imwe-Mark )
My name is Larissa. I am your code reviewer  🥰

You have implemented **all** the required changes from the previous reviewer. That is great 👌👌
I want to say, I love how you document your readme, the screenshot is a nice touch 😉
Your project is complete!  Great job 💪💪 Now how about we merge it? 🛸 🛸 
Kindly check my **optional** changes below, and if you have any questions at all, do not hesitate to mention me ❓❓❓❓
Again, congratulations! 🎉 



## Highlights

- [ ] Great documentation ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [ ] Kindly add more horizontal spacing in the **about myself** section. It is always good to have a webpage that is designed aesthetically. It would increase your chance of landing a great job 🤗🤗

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md
Comment (As1imwe-Mark) [reply README.md:1]: Thank you

**Comment by As1imwe-Mark** [comment]
@rica213 Thank you for your comments. They're a driving force

**Comment by As1imwe-Mark** [comment]
@bienvenuushindi thank you for your constructive feedback

**Pull request merged**